### PR TITLE
llvm9: switch to python3.9

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Solves (from buildmaster report):
```
llvm9_python3-9.0.1-2-x86_64.hpkg:
	requires "setuptools_python3 >= 41.4.0" of package "llvm9_python3-9.0.1-2" could not be resolved
```

File: sys-devel/llvm/llvm9-9.0.1.recipe [hunk 1]
```diff
@@ -474,7 +474,7 @@ BUILD_PREREQUIRES="
 	cmd:ld$secondaryArchSuffix
 	cmd:make
 	cmd:ninja
-	cmd:python2
+	cmd:python3
 	cmd:python$PYTHON3_VERSION
```
Comment (augiedoggie) [inline sys-devel/llvm/llvm9-9.0.1.recipe:478]: Does it need `cmd:python3` and `cmd:python$PYTHON3_VERSION`?

Comment (Begasus) [reply sys-devel/llvm/llvm9-9.0.1.recipe:478]: Good point, didn't caught my eye there ;)

File: sys-devel/llvm/patches/llvm-9.0.1.patchset [hunk 2]
```diff
@@ -120,5 +120,27 @@ index 634c16a..4ab46b9 100644
...
+
+diff --git a/utils/benchmark/src/benchmark_register.h b/utils/benchmark/src/benchmark_register.h
+index 0705e21..c731ca5 100644
+--- a/utils/benchmark/src/benchmark_register.h
++++ b/utils/benchmark/src/benchmark_register.h
+@@ -4,6 +4,7 @@
+ #include <vector>
+ 
+ #include "check.h"
++#include <limits>
```
Comment (jmairboeck) [inline sys-devel/llvm/patches/llvm-9.0.1.patchset:140]: I would sort this further up, together with `<vector>` (they both are standard library headers).

Comment (Begasus) [reply sys-devel/llvm/patches/llvm-9.0.1.patchset:140]: Fixed :)

**Pull request merged**